.github/workflows/build.yml

```diff
@@ -454,6 +518,28 @@ jobs:
         submodules: recursive
         fetch-depth: 0
 
+    # - uses: ruby/setup-ruby@v1
+    #   with:
+    #     ruby-version: 2.6
+
+    - name: Install Host toolchain
+      run: |
+        apt-get update
+        apt-get install --assume-yes build-essential ruby-full python3
+
+    - name: Recreate standalone NDK toolchains
+      run: |
+        for ARCH in arm arm64 x86 x86_64; do \
+          API=24; \
+          SYSROOT=$ANDROID_NDK_HOME/platforms/android-$API/arch-$ARCH; \
+          python3 "$ANDROID_NDK_HOME/build/tools/make_standalone_toolchain.py" \
```

Comment:
I'm going to move this part to the Docker initialization part after merging the PR